### PR TITLE
fix(deisctl): collapse component groups

### PR DESCRIPTION
**Pull request by krancour**
Addresses #4191 

Additionally, this PR aims to increase overall consistency in the the output of `deisctl install / start / stop / uninstall` commands across `platform / k8s / mesos / swarm` targets.

I have tested all of this manually since integration tests don't currently cover tech previews.

### CODE REVIEW

File: deisctl/cmd/cmd.go [hunk 1]
```diff
@@ -179,7 +179,7 @@ func startDefaultServices(b backend.Backend, stateless bool, wg *sync.WaitGroup,
 	b.Start([]string{"publisher"}, wg, out, err)
 	wg.Wait()
 
-	fmt.Fprintln(out, "Routing mesh...")
+	fmt.Fprintln(out, "Router mesh...")
```
Comment (krancour) [inline deisctl/cmd/cmd.go]: This change brings the output in-line with the nomenclature used in the docs, where we always use "router mesh" and not "routing mesh."

**Comment by mboersma** [comment]
Here is what this looks like for `deisctl install`:
```console
$ deisctl install k8s
● ▴ ■
■ ● ▴ Installing K8s...
▴ ■ ●
K8s control plane...
deis-kube-apiserver.service: loaded
deis-kube-controller-manager.service: loaded
deis-kube-scheduler.service: loaded
K8s data plane...
deis-kube-kubelet.service: loaded
K8s router mesh...
deis-kube-proxy.service: loaded
Done.

Please run `deisctl start k8s` to start K8s.
$ deisctl install mesos
● ▴ ■
■ ● ▴ Installing Mesos/Marathon...
▴ ■ ●
Mesos/Marathon control plane...
deis-zookeeper.service: loaded
deis-mesos-master.service: loaded
deis-mesos-marathon.service: loaded
Mesos/Marathon data plane...
deis-mesos-slave.service: loaded
Done.
 
Please run `deisctl start mesos` to boot up Mesos.
$ deisctl install swarm
● ▴ ■
■ ● ▴ Installing Swarm...
▴ ■ ●
Swarm control plane...
deis-swarm-manager.service: loaded
Swarm data plane...
deis-swarm-node.service: loaded
Done.
 
Please run `deisctl start swarm` to start swarm.
```

**Comment by mboersma** [comment]
LGTM. These groupings are more logical IMHO.

**Comment by carmstrong** [comment]
Code LGTM. :ship:

**Pull request merged**